lib/matplotlib/figure.py

```diff
@@ -1727,6 +1726,18 @@ def legend(self, *args, **kwargs):
         Parameters
         ----------
 
+        outside: bool or string
+            If ``constrained_layout=True``, then try and place legend outside
+            axes listed in *axs*, or highest-level gridspec if axs is empty.
+            Note, "center" and "best" options to *loc* do not work with
+            ``outside=True``.  The corner values of *loc* (i.e. "upper right")
+            will default to a horizontal layout of the legend, but this can
+            be changed by specifying a string
+            ``outside="vertical", loc="upper right"``.
+
+        axs :  sequence of `~.axes.Axes`
```

Comment:
does this need to be `axs` instead of `axes`?

Reply:
axs would be standard I think (axes is not great as it suggests a single Axes...)

Reply:
I guess `colorbar` uses `ax`.  Since this is supposed to be in parallel to that maybe I should change to that?

Reply:
sure, sounds fair

Reply:
ya that sounds good.